DrewBradley/rancid-tomatillos
# Feature/create header component

### What does this PR do?
- This PR adds a header component which contains a custom logo, name of our app, and input to search movies by title. No functionality is attached yet, but component is styled, rendered, and responsive.

### What does it fix?
- Nothing to fix.

### Is this a feature or a fix?
- This is the beginning of a feature. It's displayed on the DOM but needs functionality.

### Where should the reviewer start?
- Clone this repo down to your machine. Run npm install then run npm start. 


### How should this be tested?
- Once the page is loaded, verify that there is a header component containing the app name of "Grouch Potato" as well as a custom image and input section. Try resizing the screen to check out the break points.
